src/main/java/com/sprint/mission/discodeit/controller/BinaryContentController.java

```diff
@@ -0,0 +1,33 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.entity.BinaryContent;
+import com.sprint.mission.discodeit.service.BinaryContentService;
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/binary-content")
+public class BinaryContentController {
+
+  private final BinaryContentService binaryContentService;
+
+  @GetMapping("/{binaryContentId}")
+  public ResponseEntity<BinaryContent> getBinaryContent(
+      @PathVariable UUID binaryContentId) {
+    return ResponseEntity.ok(binaryContentService.find(binaryContentId));
+  }
+
+  @GetMapping("/batch")
```

Comment:
다건 조회시 `/batch`는 url에서 제거해도 될 것 같아요.  
```suggestion
  @GetMapping("")
```